pages/account.js

```diff
@@ -0,0 +1,207 @@
+import React, {Component} from 'react';
+import NavbarBootstrap from '../components/NavbarBootstrap';
+import Container from 'react-bootstrap/Container';
+import Card from 'react-bootstrap/Card';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Nav from 'react-bootstrap/Nav';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import CarRow from '../components/CarRow';
+import axios from 'axios';
+
+const host = 'http://localhost';
+const port = 3001;
+
+class Account extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isAuthenticated: false,
+            name: '',
+            email: '',
+            failedProfileUpdate: false,
+            succeededProfileUpdate: false,
+        };
+    }
+
+    componentDidMount = () => {
+        const authToken = window.localStorage.getItem('auth_token');
+
+        if (!authToken) {
```

Comment:
change this, we dont need this check since `isAuthenticated` is already false

just do the stuff in the else by saying
```
if (authToken){}
```